assets/xml/overlays/ovl_Obj_Jgame_Light.xml

```diff
@@ -0,0 +1,12 @@
+
+<Root>
+    <File Name="ovl_Obj_Jgame_Light" BaseAddress="0x80C152F0" RangeStart="0x8FC" RangeEnd="0xE40">
+        <!-- @bug: Based on the DList, this should be an invalid i16 -->
+        <Texture Name="D_80C15BEC" OutName="D_80C15BEC" Format="ia16" Width="16" Height="16" Offset="0x8FC"/>
+        <!-- @bug: Based on the DList, this should be an invalid i16 -->
+        <Texture Name="D_80C15DEC" OutName="D_80C15DEC" Format="ia16" Width="16" Height="16" Offset="0xAFC"/>
+        <DList Name="D_80C16030" Offset="0xD40"/>
+        <DList Name="D_80C160B0" Offset="0xDC0"/>
```

Comment:
You can name all these, we worked out that one is the O (correct) and the other the X (incorrect). And even if you weren't naming them they should still follow the scheme of suffixes (you can mark them as static, there's an xml option to, and prefix them with `s` rather than `g`).